packs/kingmaker-bestiary/akuzhail.json

```diff
@@ -228,7 +228,7 @@
                 "category": "defensive",
                 "deathNote": true,
                 "description": {
-                    "value": "<p>When Akuzhail dies, its body explodes in a riot of seething vegetation and raw life. All living creatures within a @Template[emanation|distance:60]{60-foot radius} are restored @Damage[12d6[healing]] Hit Points from this energy. A fully grown bloom of Lamashtu emerges from the carcass, but it can take no actions on this round. Next round, this bloom of Lamashtu enters combat at the same point in the turn during which Akuzhail perished on the previous round, but the bloom is @UUID[Compendium.pf2e.conditionitems.Item.Slowed]{Slowed 1}. At the end of the bloom's turn, it attempts a @Check[flat|dc:11]; on a success it loses the slowed 1 condition at the start of its next turn.</p>"
+                    "value": "<p>When Akuzhail dies, its body explodes in a riot of seething vegetation and raw life. All living creatures within a @Template[emanation|distance:60] are restored @Damage[12d6[healing]] Hit Points from this energy. A fully grown bloom of Lamashtu emerges from the carcass, but it can take no actions on this round. Next round, this bloom of Lamashtu enters combat at the same point in the turn during which Akuzhail perished on the previous round, but the bloom is @UUID[Compendium.pf2e.conditionitems.Item.Slowed]{Slowed 1}. At the end of the bloom's turn, it attempts a @Check[flat|dc:11]; on a success it loses the slowed 1 condition at the start of its next turn.</p>"
```

Comment:
```suggestion
                    "value": "<p>When Akuzhail dies, its body explodes in a riot of seething vegetation and raw life. All living creatures within a @Template[emanation|distance:60]{60-foot radius} are restored @Damage[12d6[healing]] Hit Points from this energy. A fully grown bloom of Lamashtu emerges from the carcass, but it can take no actions on this round. Next round, this bloom of Lamashtu enters combat at the same point in the turn during which Akuzhail perished on the previous round, but the bloom is @UUID[Compendium.pf2e.conditionitems.Item.Slowed]{Slowed 1}. At the end of the bloom's turn, it attempts a @Check[flat|dc:11]; on a success it loses the slowed 1 condition at the start of its next turn.</p>"
```